source/common/upstream/eds.cc

```diff
@@ -95,6 +95,9 @@ void EdsClusterImpl::onConfigUpdate(const ResourceVector& resources, const std::
 
   // Track whether we rebuilt any LB structures.
   bool cluster_rebuilt = false;
+
+  // Loop over all priorities present in the config. This will either update an existing
+  // HostSet or create a new one.
```

Comment:
I think 
// Loop over existing priorities not present in the config.  This will empty out any priorities the config update did not refer to
might be a bit easier to read

Reply:
I like it! Will update